src/Illuminate/Auth/Notifications/ResetPassword.php

```diff
@@ -56,9 +56,9 @@ public function toMail($notifiable)
         }
 
         return (new MailMessage)
-            ->line('You are receiving this email because we received a password reset request for your account.')
-            ->action('Reset Password', url(config('app.url').route('password.reset', $this->token, false)))
-            ->line('If you did not request a password reset, no further action is required.');
+            ->line(__('You are receiving this email because we received a password reset request for your account.'))
```

Comment:
@countnoobula Taylor mentioned that he'd accept a pull-request if the json translation features was used instead of translation keys (see https://github.com/laravel/ideas/issues/409).

Reply:
The auth component can't use helpers defined in the foundation package.

Reply:
I actually tested this before submitting the PR and it does work as expected.

Reply:
> I actually tested this before submitting the PR and it does work as expected.

Did you test using the auth component outside of the framework?

Reply:
This notification won't work without the rest of the framework anyway, will it? It can't be used in isolation.

Reply:
It should work without the foundation package, such as in Lumen. If there is genuinely code in here that doesn't work, even with every illuminate package installed, then it is in the wrong place, and should be in foundation.

Reply:
Correct me if I'm wrong: Lumen does not use the ResetPassword notification. And if one wants to use it they need to pull in all sorts of packages anyway. If Foundation is the one package they shouldn't need to pull in, then the translation helper should live somewhere else, shouldn't it?

Reply:
@GrahamCampbell Ahh.. I see. Foundation is not a seperate package that can be pulled in. Shame on me.

That said.. all the other password reset stuff is already in foundation.. so this should probably go there, too, right?